app/controllers/organizations_controller.rb

```diff
@@ -125,6 +126,23 @@ def set_users_github_organizations
   end
   # rubocop:enable AbcSize
 
+  # Check if the current user has any organizations with admin privilege, if so add the user to the corresponding
+  # classroom automatically.
+  def auto_scan_organizations_access
+    @users_github_organizations.each do |organization|
+      organization = organization[:classroom]
+      next unless organization.present?
+      next if organization.users.include?(current_user)
+      create_user_organization_access(organization)
+    end
+  end
```

Comment:
@johndbritton I think your line needs to be

```diff
- if organization.present? && organization.users.include?(current_user)
+ if organization.present? && !organization.users.include?(current_user)
```

What do you think about this?

```ruby
def auto_scan_organizations_access
  @users_github_organizations.each do |organization|
    classroom = organization[:classroom]
    if classroom.present? && !classroom.users.include?(current_user)
      create_user_organization_access(classroom)
    end
  end
end
```